packages/ffe-cards/less/components.less

```diff
@@ -22,9 +22,6 @@
 
     &__title {
         &:extend(.ffe-h5);
-        /* stylelint-disable-next-line block-no-empty */
-        .native &:extend(.native .ffe-h5) {
```

Comment:
Fjernes denne fordi det blir dobbelt opp? eller hva skjer her?

Reply:
Den kan fjernes fordi ikke lengre finnes en regel for `.native .ffe-h5`. Reglen kom fra `ffe-core/less/typography.less` og ble generert fra [.ffe-fontsize-h5() ](https://github.com/SpareBank1/designsystem/pull/1383/files#diff-aefe55be50fcbc11c52304492d183b9faa3994969fab342efc493def56119cffL60).